### DISCUSSION TITLE
Add `sed` support

### DISCUSSION THREAD

**Discussion by gunta** [google/zx#672] [General]
Since sed is such a common command, it would be nice to have a cross-platform command for that.

**Comment by nichtich** [comment]
Could you explain with an example and references?

**Comment by gunta** [comment]
Shelljs has a nice sample of it working [here](https://github.com/shelljs/shelljs)
It is such a common case that it's even featured as the first sample.

Something like this would work:
```javascript
await sed('-i', /^.*REMOVE_THIS_LINE.*$/, '', "file.txt")
```

**Comment by nichtich** [comment]
I don't see why sed is more relevant than e.g. awk or perl. Why not directly use shelljs?
